mlir/lib/Dialect/AMDGPU/IR/AMDGPUDialect.cpp

```diff
@@ -326,6 +326,68 @@ LogicalResult MFMAOp::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// DPPOp
+//===----------------------------------------------------------------------===//
+LogicalResult DPPOp::verify() {
+  Type srcType = getSrc().getType();
+  if (srcType.getIntOrFloatBitWidth() > 32) {
+    return emitOpError("integer and floating point types larger than 32 bits "
+                       "are not supported");
```

Comment:
64-bit is actually supported already, and PRs are up to extend this to any legal-ish type

Reply:
I was not aware of that. Can you point me to the change?

Reply:
i64 has been supported for a while, and it's directly supported on some MI targets too. #91190 handled it for f64